go/flags/endtoend/vtctld.txt

```diff
@@ -128,9 +116,6 @@ Usage of vtctld:
       --tablet_manager_grpc_server_name string                           the server name to use to validate server certificate
       --tablet_manager_protocol string                                   Protocol to use to make tabletmanager RPCs to vttablets. (default "grpc")
       --tablet_protocol string                                           Protocol to use to make queryservice RPCs to vttablets. (default "grpc")
-      --tablet_refresh_interval duration                                 Tablet refresh interval. (default 1m0s)
-      --tablet_refresh_known_tablets                                     Whether to reload the tablet's address/port map from topo in case they change. (default true)
```

Comment:
these are the ones I wanted @frouioui to check on.
I believe they are used if `enable_realtime_stats` is set. So we can deprecate them but not delete them.

Reply:
From my observations, you are right, in `vtctld` those flags are used **only** if the `--enable_realtime_stats`. However those two flags are still used by `vtgate`.